src/components/Clock.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export class Clock extends React.Component {
+  state = {
+    date: new Date(),
+  }
+
+  componentDidMount() {
+    this.state.timerID = setInterval(
+      () => this.setState({
+        date: new Date(),
+      }), 1000,
```

Comment:
```suggestion
    this.timerID = setInterval(
      () => this.setState({
        date: new Date(),
      }), 1000,
```
You should add smth to state using `setState` method (not just directly `this.state.timerId =...`, as you have written.
Also, there is no need to write setInterval to the stat. You may just use `this` for that purpose